tests/test_confidence.py

```diff
@@ -79,3 +79,16 @@ def some_prop():
 """
     check_min_confidence(code, 50, {'some_prop': dc})
     check_min_confidence(code, 100, {})
+
+
+def test_confidence_async_def():
```

Comment:
This should use the skip_if_not_has_async decoreator, I think.

Reply:
Oh! sorry, I forgot :p. Thanks!